simtools/layout/layout_array.py

```diff
@@ -563,6 +567,9 @@ def read_telescope_list_file(telescope_list_file):
             _logger.error(f"Error reading list of array elements from {telescope_list_file}")
             raise
         _logger.info(f"Reading array elements from {telescope_list_file}")
+        if np.any([col.dtype == np.float32 for col in table.columns.values()]):
```

Comment:
This was a conscious decisions, as right now we test all columns not be float32 (e.g., also the telescope names, geocode, counters). So we allow for string and int. I do think this is enough in this case.

Alternatively: would have to have a list of all allowed axes and test them. This would be a lot more code, not sure if this is needed.

Reply:
OK, this solution should be fine then (perhaps will change in the future with a full schema, we'll see if it's necessary).